isyfact-standards-doc/src/docs/antora/modules/isy-security/pages/konzept/inhalt.adoc

```diff
@@ -316,47 +316,18 @@ image::konzept/authorization-code-flow.dn.svg[align="center"]
-Der _Client-Credential-Flow_ dient dabei der Authentifizierung eines technischen Nutzers und über den _Resource-Owner-Password-Credentials-Flow_ erfolgt die Authentifizierung eines Anwenders.
-Mit validen Authentifizierungsdaten und erfolgreicher Authentifizierung liefert der IAM-Service dann ein valides, neu erstelltes Access-Token an die externe Anwendung zurück.
-
-Mit Erhalt eines gültigen Access-Tokens kann nun die externe Anwendung ihre fachlichen Aufrufe zusammen mit dem Token über das _Service-Gateway_ an das eigentliche Anwendungssystem senden.
-Denn in dem Access-Token sind alle notwendigen Informationen enthalten, um im _Service-Gateway_ eine Validierung auf eine zuvor stattgefundene und gültige Authentifizierung, sowie eine Prüfung auf eine valide Autorisierung des Requests vorzunehmen.
-So sind neben einer Gültigkeitsdauer auch die für den Benutzer autorisierten Rollen in dem Token enthalten, sodass im Anwendungssystem die Ausführungsberechtigung des Benutzers verifiziert und sichergestellt werden kann.
-
-Das nachfolgende Sequenzdiagramm zeigt noch einmal schematisch den logischen Ablauf der Authentifizierung und Autorisierung im Zusammenspiel der System-Komponenten und dem Authentication-Gateway:
-
-.Sequenzdiagramm Authentifizierung am Authentication-Gateway
-[id="image-sequenzdiagramm-sgw-plus-agw",reftext="{figure-caption} {counter:figures}"]
-image::konzept/Squenzdiagramm_ServiceGateway_plus_AuthGateway.dn.svg[align="center"]
-
+Anfragen an ein IsyFact-Anwendungssystem, die von Außen eingehen, müssen über den IAM-Service authentifiziert und zur Nutzung des aufgerufenen IsyFact-Service autorisiert sein.
+Um das zu gewährleisten, ist in der Informations- und Dienstezone ein _Service-Gateway (SGW)_ zwischen der externen Anwendung und dem internen IsyFact-Service geschaltet.
+Das Service-Gateway nimmt die fachliche Anfrage von Außen entgegen, prüft die Berechtigung und leitet die Anfrage weiter an den internen IsyFact-Service.
 
 [[aufruf-service-gateway-ohne-authentication-gateway]]
 ===== Authentifizierung über das SGW am IAM-Service
 
+Aktuell erfolgt die Authentifizierung von Anfragen externer Anwendungen direkt im Service-Gateway.
+Dafür müssen bei jeder Anfrage auch die Authentifizierungsdaten mitgeschickt werden.
+Das Service-Gateway lässt die Authentifizierungsdaten durch den IAM-Service prüfen und leitet erst nach dessen OK die fachliche Anfrage an das Anwendungssystem weiter - jetzt zusammen mit dem Access-Token.
+
 Die nachfolgende Abbildung zeigt die System-Architektur, die bei Aufrufen von externen Anwendungen für die Authentifizierung ohne _Authentication-Gateway_ zum Einsatz kommt.
-Diese Authentifzierungs- und Autorisierungs-Variante wird für eine einfache Migration von externen Anwendungen auf die neue `IsyFact-3` zusätzlich beibehalten, da hierdurch die Umstellungsaufwände reduziert werden.
```

Comment:
Warum soll diese Information gelöscht werden?
Sie ist m.E. eine sehr wichtige Information für eine Externe Anwendung, die auf IsyFact-3.0 migrieren möchte. Denn das bedeutet, dass trotz intern in IsyFact umgestellter Authentifizierung von isy-sicherheit auf isy-security die externe Anwendung ihre bisherigen Requests an eine IsyFact-Nachbaranwendung nicht aufwändig anpassen muss.

Reply:
Diesen Satz habe ich gelöscht, da aktuell (IF 3.0) keine äusserliche Änderung in der Art der Authentifizierung gemacht wird. Die Einführung des Auth-GW erfolgt erst später und erst dann kann und sollte beschrieben werden, wie eine evtl. Migration aussehen kann.